### PR TITLE
Checkbox / Radio Styling 

### PR DESCRIPTION
**Pull request by joebell93**
## Description
Provides error styling for checkbox and radio indicator borders

## Related Issue
https://github.com/sky-uk/toolkit-ui/issues/33

## How Has This Been Tested?
Visually in all browsers

## Screenshots (if appropriate)
screen shot 2016-10-03 at 15 03 49

## Types of Changes
- [x] Bug fix (non-breaking change which fixes an issue)

## Browser Support
- [x] Chrome
- [x] Edge
- [x] Firefox
- [x] IE9
- [x] IE10
- [x] IE11
- [x] Opera
- [x] Safari
- [x] Mobile Devices

## Checklist
- [x] New functions and mixins have appropriate tests.
- [x] All new and existing tests passed.
- [x] I have added instructions on how to test my changes.
- [x] Package version updated.
- [x] CHANGELOG.md updated.

### CODE REVIEW

File: components/_forms.scss [hunk 1]
```diff
@@ -397,17 +396,20 @@ $form-animation-speed: $global-animation-speed-fast;
...
+ * Change form field styles
+ */
+.c-form-date,
 .c-form-input,
-.c-form-select__dropdown,
-.c-form-checkbox__faux {
+.c-form-select__dropdown {
 
   .is-error &,
   &.is-error,
```
Comment (skitson) [inline components/_forms.scss]: Do we need both? Are there instances where the `is-error` class is only added to the `c-form-select__dropdown` container and not on the parent and vice versa? I'm presuming there are?

Comment (joebell93) [inline components/_forms.scss]: Just provides a little extra flexibility, so we can encourage people to put `is-error` on `c-form-list__item` to modify the label too

**Comment by joebell93** [comment]
Cheers! @skitson @MrDinsdale

File: components/_forms.scss (hunk 1 above)
Comment (joebell93) [inline components/_forms.scss]: Worth investigating though going forwards

**Pull request merged**